core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -287,19 +319,33 @@ public boolean testClassLoad(final String className, final String asTypeName)
   }
 
   @Override
+  @Deprecated
   public List<ActiveCompaction> getActiveCompactions(String server)
       throws AccumuloException, AccumuloSecurityException {
-    final var serverHostAndPort = HostAndPort.fromString(server);
 
+    ServerId si = context.getServerIdResolver().resolveCompactor(server);
+    if (si == null) {
+      si = context.getServerIdResolver().resolveTabletServer(server);
+    }
+    if (si == null) {
+      return new ArrayList<>();
+    }
+    return getActiveCompactions(si);
+  }
+
+  @Override
+  public List<ActiveCompaction> getActiveCompactions(ServerId server)
+      throws AccumuloException, AccumuloSecurityException {
+
```

Comment:
This could have a Preconditions check to check the server type like getActiveScans does.

Reply:
Suggestion implemented in ebd6763